### PR TITLE
JavaScript capstone project :  API-based webapp

### PR DESCRIPTION
**Pull request by isaacpitwa**
### In this PR:
1. Built these interfaces:
   -   The home page.
   -   The comments popup.
2. When the page loads, the webapp retrieves data from:
   - The selected API and shows the list of items on screen.
   - The Involvement API to show the item likes.
3. The page makes only 2 requests:
   - One to the base API.
   - And one to the Involvement API.
4. When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
5. When the user clicks on the "Comments" button, the Comments popup appears.
6. When the popup loads, the webapp retrieves data from:
    - The selected API and shows details about the selected item.
    - The Involvement API to show the item comments.
7. When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.
8. We have counters in all the interfaces that show:
   - The number of items (home).
   - The number of comments (comments popup).
9. Created specific functions to calculate the number of items (home) and the number of comments (comments popup). These count functions are covered with unit tests using Jest.

### CODE REVIEW

File: src/modules/display.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    event.preventDefault();
+    submitBtn.innerHTML = '<i class="fa fa-spinner"></i> Loading...';
+    addMovieComment({ item_id: id, username: nameInput.value, comment: commentInput.value }).then(
+      () => {
+        submitBtn.innerText = 'comment';
+        form.reset();
+        fetchMovieComments(id, contentSide, true);
+      },
+      () => {
+        // alert(`${error}`);
```
Comment (Dipeshtwis) [inline src/modules/display.js:105]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

File: src/modules/functions.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { getMovies } from './API.js';
+import display from './display.js';
+
+const createCards = async () => {
+  // const movies = ['1', '6', '3', '4', '7', '8', '9', '10', '11', '15', '25', '30'];
```
Comment (Dipeshtwis) [inline src/modules/functions.js:5]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

File: README.md [hunk 3]
```diff
@@ -1 +1,74 @@
...
+
+- Enter in to the folder:
+`cd Capstone-2`
+
+- Run `npm i` to install all dependencies
+- Run `npm run` build to bundle the project with webpack, and
+- Run `npm start` to launch the application
+
+- Open the folder with a code editor (VS Code preferred)
+
```
Comment (Dipeshtwis) [inline README.md:38]: - [ ] I would recommend adding the command to test the application as well. Hint: `npm test`.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @isaacpitwa & @Juanse7793, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 3 above)
Comment (Juanse7793) [reply README.md:38]: thank you, it's done :+1:

**Review by hiromataba — APPROVED** [review]
# Status: Approved 🎉 🎉 
Hi 👋 👋 @isaacpitwa 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**